.travis.yml

```diff
@@ -11,5 +11,8 @@ install:
   # - yarn bootstrap
   - travis_retry make ci-install
 script:
+  - export SERVER=https://node.cryptape.com
```

Comment:
should  not use testnet for testing in CI

Reply:
Ok, what other nodes can I connect to? Or skip this test?

Reply:
@Keith-CY how to do system testing before?

Reply:
All tests run locally by the `pre-commit` hook and only the tests of `utils`, `formatters`, `address` modules run in the CI.

Reply:
We've planned to run a docker in the CI, but it's suspended.

Reply:
@rainchen  We configure the CI environment parameters at https://travis-ci.org.
A commit with a CI error does not seem to read the environment parameter, PRIVATE_KEY is undefined.

Reply:
using ENV in travisCI is an acceptable solution before we can have an auto testing env with docker